scripts/busybox-ping-test/alcor_services.ini

```diff
@@ -1,20 +1,20 @@
 [services]
 ignite                 = {"name":"ignite", "port":10800, "path":"/lib/ignite.Dockerfile"}
 vpc_manager            = {"name":"vpm",    "port":9001}
```

Comment:
How about VPC mgr? One admin API would make every API for the same manager not tested through API GW :-(

Reply:
I will look into this and make the change.